harness/determined/pytorch/_pytorch_context.py

```diff
@@ -116,6 +132,50 @@ def get_per_slot_batch_size(self) -> int:
         """
         return self._per_slot_batch_size
 
+    def get_experiment_config(self) -> Dict[str, Any]:
```

Comment:
you are missing at least the following TrialContext APIs (which are no longer inherited):
- get_data_config
- get_experiment_id
- get_trial_id

Reply:
fixed